.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
```

Comment:
I guess this means that once we have a PR every push will be 2 identical builds to wait for?

I saw you justifying we want this for external contributors - it'd be good to work out if we can avoid the double build situation before propagating across our apps.

Reply:
Indeed - I think we can just use `push` here.

Reply:
I'll switch this to use `pull_request` and the `master` branch, as discussed in the RFC. Then we can see how we go.